src/containers/shared/components/Transaction/Clawback/Simple.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { ClawbackInstructions } from './types'
+import { Account } from '../../Account'
+import { Amount } from '../../Amount'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps<ClawbackInstructions>) => {
+  const { account: issuer, amount, holder } = data.instructions
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <SimpleRow label={t('issuer')} data-test="issuer">
```

Comment:
This seems redundant since it is already in the amount.

Reply:
I don’t think it would hurt tho. It’s really minimal, and I put there to just make it dummy-proof

Reply:
Isn't the issuer also the sender of the transaction? Seems extra unnecessary

Reply:
fixed